### ISSUE TITLE
Running a editorscript with code selected performs an unintentional cut of the code

### ISSUE THREAD

**Issue by balloonpopper** [godotengine/godot#57377]
### Godot version

3.4.2 stable

### System information

Windows 10

### Issue description

When writing gdscript in the code editor, if you have text selected then run the code using the hot key (cntl-shift-x), instead of just running the code, it also performs the cut operation (i.e. ctrl-x) 

### Steps to reproduce

Create a node on any project and add the following script to it

```
tool
extends EditorScript
func _run() -> void:
	print("A")
	print("B")
```

Highlight the "print("B")" line. 
Hit ctrl-shift-x
You will see "A" printed, but the line with "B" will be removed from your script.

**Comment by Ev01** [comment]
This seems to be fixed in 4.0 alpha 1 (windows 10)

**Comment by EricEzaM** [comment]
See https://github.com/godotengine/godot/issues/52861
